cppcache/integration-test/testThinClientRemoteQueryTimeout.cpp

```diff
@@ -426,7 +426,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, verifyNegativeValueTimeout)
     try {
       LOG("Task::verifyNegativeValueTimeout - EXECUTE 1 START");
 
-      results = qry->execute(-3);
```

Comment:
We support negatives? Interseting...

Reply:
Ah, I think I got confused by multiple log statements afterwards which don't get run unless it fails.